homeassistant/components/here_travel_time/sensor.py

```diff
@@ -172,7 +173,12 @@
         name, origin, destination, origin_entity_id, destination_entity_id, here_data
     )
 
-    async_add_entities([sensor], True)
+    def delayed_sensor_update(event):
+        """Update sensor after homeassistant started."""
+        sensor.async_schedule_update_ha_state(True)
+
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, delayed_sensor_update)
```

Comment:
Move this inside the entity in `async_added_to_hass`.

Reply:
Two hours searching the other components for best practices and still missed something 😅 
Thank you!